var/spack/repos/builtin/packages/zlib/package.py

```diff
@@ -34,6 +32,8 @@ class Zlib(Package):
     patch('w_patch.patch', when="@1.2.11%cce")
     patch('configure-cc.patch', when='@1.2.12')
 
+    depends_on('cmake@3.15:')
```

Comment:
This is unlikely to work on Linux because CMake itself depends on zlib (transitively). This package will likely need to be updated to only depend-on/run CMake on Windows.

Reply:
I added a conditional to only enforce the CMake dependency on Windows

Reply:
I don't see that here yet. Also, if you inherit `CMakePackage`, that will automatically add a `CMake` dependency (which you cannot override with `depends_on` here).